.checkstyle/checkstyle.xml

```diff
@@ -66,7 +66,7 @@
 
         <!-- dependencies and import control -->
         <module name="ImportControl">
-            <property name="file" value="${importControlFile}"/>
+            <property name="file" value=".checkstyle/import-control.xml"/>
```

Comment:
FYI: The change here and on line 136 does not seem to work and breaks the build.

Reply:
Fixed :+1: